docs/content/documentation/deployment/github-pages.md

```diff
@@ -4,8 +4,7 @@ weight = 30
 +++
 
 By default, GitHub Pages uses Jekyll (A ruby based static site generator),
-but you can use whatever you want provided you have an `index.html` file in the root of a branch called `gh-pages`.
-That branch name can also be manually changed in the settings of a repository.
+but you can also publish any generated files provided you have an `index.html` file in the root of a branch called `gh-pages` or `master`, in addition you can also publish from a `docs` directory in your repository. That branch name can also be manually changed in the settings of a repository. **However** this only applies to publishing in a custom domain, i.e. if you want to publish to a GitHub provided web service under the `github.io` domain, you can **only** use the `master` branch of your repository as explained [here](https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages), so we will focus on the method which will work regardless of the domain.
```

Comment:
is the `docs` directory fixed or can you change that to `public`?

Reply:
As per the linked site it is fixed to the /docs directory of the master branch.

Reply:
Is there any way to change the zola build to output into docs instead of public? Possibly adding a field to config.toml?